src/ui/search.tsx

```diff
@@ -113,141 +117,143 @@ export function Search() {
-										</A>
-									</div>
-								</div>
-							</Suspense>
+													return (
+														<li ref={ref => setResultRefs(resultRefs => {
+															const newRefs = [...resultRefs];
+															newRefs[itemIndex] = ref;
+															return newRefs;
+														})} role="option" aria-selected={itemIndex === active() ? "true" : "false"}>
```

Comment:
Giving a `scroll-margin` to search result items when auto-scroll can create some space like the paddings. Currently, they are adjacent to the top or bottom side of the search results section.

```suggestion
														})} class="scroll-my-2" role="option" aria-selected={itemIndex === active() ? "true" : "false"}>
```

Before:

https://github.com/user-attachments/assets/83b519ef-2977-4393-9cbc-bd6faa50e91e

After:

https://github.com/user-attachments/assets/70814acd-1fad-4576-bfc7-da9ad1cde6b3